dll/win32/shell32/lang/cs-CZ.rc

```diff
@@ -925,6 +925,7 @@ BEGIN
     IDS_NONE "(None)"
 
     /* Friendly File Type Names */
+    IDS_FILE "Soubor"
```

Comment:
It is best to leave the resource string left untranslated for other locales and let other maintainers translate it, whereas you can just simply translate it in your own native language(s).

Reply:
I see, This time I diverted from IDS_ANY_FILE.